game/lib/gameplay/gameplay_page.dart

```diff
@@ -2,9 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter/src/foundation/key.dart';
 import 'package:flutter/src/widgets/framework.dart';
 import 'package:flutter/src/widgets/placeholder.dart';
+import 'package:velocity_x/velocity_x.dart';
 
 import 'gameplay_map.dart';
 
+// Returns widget that fills based on tasks and tasks_completed.
 List<Widget> generate_progress(tasks, tasks_completed) {
```

Comment:
Make sure to use camelCase when writing out variable names. Also, explicitly specify the type of the arguments so it isn't dynamic.